resources/views/auth/login.blade.php

```diff
@@ -0,0 +1,51 @@
+@extends('layouts.app')
+@section('content')
+<main class="login-form">
+    <div class="cotainer">
+        <div class="row justify-content-center">
+            <div class="col-md-4">
+                @if (Session::get('error'))
+                <div class="alert alert-success">
+                    {{ Session::get('error') }}
+                </div>
+                @endif
```

Comment:
```suggestion
                @if (Session::get('error'))
                    <div class="alert alert-success">
                        {{ Session::get('error') }}
                    </div>
                @endif
```
fix tương tự các chỗ khác nha